tech-far-hub/src/components/initiative.tsx

```diff
@@ -21,11 +21,9 @@ export const Initiative = ({ children, heading, media, media_alt, destination }:
       </CardHeader>
       <CardMedia exdent>{image && <GatsbyImage image={image} alt="A stock image" />}</CardMedia>
       <div className="tfh-initiativeCard-internal">
-        <CardBody>
-          <p>{children}</p>
-        </CardBody>
+        <CardBody>{children}</CardBody>
         <CardFooter>
-          <Link to={destination} className="usa-button">
+          <Link to={destination} className="usa-button" aria-label={`View ${heading}`}>
```

Comment:
Did you use some tool to flag this missing ARIA tag? I'm using AxeDev Tool (Chrome dev tools), however I haven't seen this

Reply:
No --- a colleague brought up the accessibility problems of "View" as the link text, and I was looking for a way to get more information to screen readers and I was like "wait, this is what aria is for!"